uiowa/uiowa
# Feature: Set layout builder section container settings to "Normal"

Resolves https://github.com/uiowa/uiowa/issues/3451. 

- [x] Removed "-None-" option
- [x] Set label for "Normal" to "Normal (Default)"
- [x] Set Default setting to "Normal" if creating a new section
- [x] Update hook to set sections to "Normal" if they were previously set to "-None-"

May want to push code up to Dev first, as a test to see if batching is needed; but otherwise this is ready for testing.

# How to test

Check the labeling
 - Add a new section or edit the configuration of an existing section
 - Verify that there is no "- None -" option available in the Container settings
 - Verify that the "Normal" option is available, and that it is labelled "Normal (Default)" in the Container settings

Check the new default
 - Add a new section
 - Verify that the Container settings defaulted to "Normal (default)"

Regular update hook testing

- Note sections which currently have `- None -` container style selected, or add your own
- Run the update hook--check that the "(after)" message for both node and node_revision tables list 0
- Edit the layouts for any pages noted originally, and check that the section configurations are set to the `Normal (default)` container styles

Temp Store testing

- Checkout main and sync a site
- Edit layout(s) and add sections with the `- None -` container style selected, but do not save
- Checkout `feature_lb_container_normal_settings` and run the update hook (can visit `/update.php` in the browser to run it)
- Edit layout(s) (they should still be listed with "You have unsaved changes"--check that the newly added sections, as well as any existing sections, have properly updated to `Normal (default)` container styles
